headers/types/dungeon_mode/enums.h

```diff
@@ -1442,4 +1442,11 @@ enum fade_status_dungeon {
     DUNGEON_FADE_0x5 = 5,
 };
 
+// Used to indicate whether two monsters are allies, enemies, or neither.
+enum monster_relation {
+    RELATION_ALLY = 0,
+    RELATION_ENEMY = 1,
+    RELATION_OTHER = 2,
+};
```

Comment:
Since we now know this is actually used to determine how one pokémon should treat another, how about renaming it?

```suggestion
// Used to indicate how a given monster should target another.
enum monster_treatment {
    TREATMENT_TREAT_AS_ALLY = 0,
    TREATMENT_TREAT_AS_ENEMY = 1,
    TREATMENT_IGNORE = 2,
};
```
@AnonymousRandomPerson let me know if those values make sense, since you seem to be familiar with the function already.

Reply:
Yep, names like these make sense.